docs/api-docs/channels/channel-app-best-practices.md

```diff
@@ -1,65 +1,78 @@
 # Channel App Best Practices
 
+<!-- https://developer.bigcommerce.com/api-docs/channels/guide/channel-app-best-practices -->

```

Comment:
@markcmurphy, what's this here for?

Reply:
@uwikaiddi The /api-docs file structure and markdown file names don't totally correspond to the left-hand nav on developer.bigcommerce.com. I added code comments with the URL so someone unfamiliar with the repo can reference the live page faster (or verify they're working on the intended page). They're mostly small differences like the markdown file for the Building Channels -> Guide -> Introduction being named `channels-overview.md`. 

Should I remove them?

Reply:
@markcmurphy , no that's a good idea. Leave them, but add a bit more context as to what the information is.
Suggestion: `<!-- Dev Center URL: https://developer.bigcommerce.com/api-docs/channels/guide/channel-app-best-practices -->